### PR TITLE
Reintegrate GMM for Community Embeddings for Comparison

### PR DESCRIPTION
**Pull request by abegehr**

### CODE REVIEW

File: env.yml [hunk 1]
```diff
@@ -86,3 +86,12 @@ dependencies:
     - numpy==1.18.1
 prefix: /Users/anton/opt/miniconda3/envs/BICE
 
+
```
Comment (abegehr) [inline env.yml:89]: this file looks wrong.

**Comment by abegehr** [comment]
@Elli-cyber @tguelenman I'd be grateful for a review

File: env.yml (hunk 1 above)
Comment (Elli-cyber) [reply env.yml:89]: yea i think so too

**Comment by Elli-cyber** [comment]
All in all very well done 😘🤓

**Pull request merged**